### ISSUE TITLE
val_loss=inf

### ISSUE THREAD

**Issue by robinenrico** [qqwweee/keras-yolo3#374]
I started training with train_bottleneck.py and managed to get a fast decreasing training loss till train_los=30 for epoch=25. However, the validation loss got the value "Inf" from epoch 1. Does anyone recognise this issue and how to solve this?

<img width="651" alt="Screenshot 2019-03-17 at 20 59 58" src="https://user-images.githubusercontent.com/48655808/54497153-a7bfd600-48f7-11e9-80a7-b4c7fde2452c.png">

**Comment by robinenrico** [comment]
Seems to be a data related issue, comparable with the many val_loss=non issues reported. Issue solved by changing model.py, according to: https://github.com/qqwweee/keras-yolo3/issues/305#issuecomment-463042788
